.github/workflows/restart_recovery_suite.yml

```diff
@@ -2,6 +2,7 @@ name: Restart recovery suite
 on:
   schedule:
     - cron: '0 1 * * *'
+on: push
```

Comment:
Question for me to get a better understanding of the git workflows - why do we need the "on: push" option? I understand the above one schedules the job periodically, but am not sure what the second does?

Reply:
Just for testing purposes. It will not be merged.